bitnami/common/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/common
 type: library
-version: 2.28.0
+version: 2.29.0
```

Comment:
`appVersion` must be also updated for this special chart.

Reply:
Updated